README.md

```diff
@@ -1,10 +1,50 @@
 # ROS2 KUKA Drivers
 
-Experimental ROS2 driver for KUKA robots.
+Experimental ROS2 driver for KUKA robots. Recommended distribution is [ROS 2 Humble Hawksbill:](https://docs.ros.org/en/humble/Installation.html).
 
+> ROS2 Iron Irwini has breaking changes, thus it is not yet supported.
 
 Github CI | SonarCloud
 ------------| ---------------
 [![Build Status](https://github.com/kroshu//kuka_drivers/workflows/CI/badge.svg?branch=master)](https://github.com/kroshu/ros2_kuka_sunrise_fri_driver/actions) | [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=kroshu_kuka_drivers&metric=alert_status)](https://sonarcloud.io/dashboard?id=kroshu_kuka_drivers)
 
-Documentation of this project can be found on the repository's [Wiki](https://github.com/kroshu/kuka_drivers/wiki) page.
+# Installation
+
+Create ROS2 workspace (if already not created).
+```bash
+mkdir -p ~/ros2_ws/src
+```
+
+Clone KUKA ROS2 repositories.
+```bash
+cd ~/ros2_ws/src
+git clone https://github.com/kroshu/kuka_drivers.git
+vcs import < kuka_drivers/upstream.repos
+```
+
+Install dependencies using `rosdep`.
+```bash
+cd ~/ros2_ws
+sudo apt install python3-rosdep
+sudo rosdep init
+rosdep update
+sudo apt upgrade
+rosdep install -r --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
```

Comment:
```suggestion
rosdep install --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
```

It's only a suggestion, but I would not add `-r` to `rosdep install` command lines.

It let's `rosdep` ignore dependencies it either couldn't resolve or install and continue with "the rest".

That may sound like a good thing, but in reality it just confuses users: `rosdep` will print a summary at the end listing dependencies it couldn't install, but in my experience, as long as users "get back the command prompt", they assume the previous command was successful. Or they don't read `rosdep` output at all, because "there's so much of it".

Failing early (and loudly) is better in those cases: without `-r`, `rosdep` will only print an error and then not do anything.

If you added `-r` to avoid `rosdep` stopping because one of your dependencies doesn't have a `rosdep` key (yet), I'd suggest considering to add a `--skip-key="your_key [other_key [..]]" for now.